tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {
```

Comment:
what for we have this word here?
¯\(°_o)/¯

Reply:
which one?